packages/next-auth/src/providers/yandex.ts

```diff
@@ -0,0 +1,58 @@
+import { OAuthConfig, OAuthUserConfig } from "."
```

Comment:
This file change should be reverted and optionally re-opened against the `v4` branch.

File is here: https://github.com/nextauthjs/next-auth/blob/v4/packages/next-auth/src/providers/yandex.js